src/parse/parser.rs

```diff
@@ -1231,12 +1231,9 @@ impl<'help, 'app> Parser<'help, 'app> {
                     "Parser::parse_long_arg: Found an opt with value '{:?}'",
                     &val
                 );
-                self.parse_opt(&val, opt, matcher, trailing_values)
+                self.parse_opt(val, opt, matcher, trailing_values)
             } else if let Some(rest) = val {
-                debug!(
-                    "Parser::parse_long_arg: Got invalid literal `{:?}`",
-                    rest.to_os_string()
```

Comment:
This is a great idea to improve the `Debug` output. I'll incorporate this into `os_str_bytes` so you don't need this hack.